subsys/bluetooth/audio/Kconfig.baps

```diff
@@ -7,7 +7,7 @@
 #
 
 config BT_AUDIO_UNICAST
-	bool
+	bool "Bluetooth Unicast"
```

Comment:
`BT_AUDIO_UNICAST` is intentionally left unselectable by the application. The application should use `BT_AUDIO_UNICAST_SERVER` and/or `BT_AUDIO_UNICAST_CLIENT`

Reply:
And when that is the problem? The application cannot use UNICAST_CLIENT or SERVER, as it drags in too many dependencies. We need a configuration without this overhead.

Reply:
Personally I don't get why we force an application to use "experimental" BT_AUDIO_UNICAST_SERVER, which forces things like BT_SMP_SC_PAIR_ONLY?

Reply:
The Unicast Server and Unicast Client configuration are based on the requirements from the Basic Audio Profile (BAP). If you want to be compliant, these need to be enabled. 

Something like EATT (and thus dynamic channels) could be removed, as they are not strictly required. 

Just to avoid any confusion here: Are you trying to just enable ISO?

Reply:
Previously we had to enable BT_AUDIO_UNICAST to get ISO functionality. That may have changed - I think that's where I went wrong. I was focused on getting the same configs activated as for our previous integration point.

Reply:
My mistake, sorry.